testcases/PowerNVDump.py

```diff
@@ -414,5 +488,6 @@ def crash_suite():
     s.addTest(SkirootKernelCrash())
     s.addTest(OPALCrash_MPIPL())
     s.addTest(KernelCrash_FadumpEnable())
+    s.addTest(SBECrash_MPIPL())
```

Comment:
Lets not include this test in default suite..as we already have enough test cases to cover MPIPL from OPAL/kernel point of view. 

This test case is useful for hostbot and SBE folks. Let them call this test cases directly.

-Vasant

Reply:
Sure